### PR TITLE
Obliterates bad syntax

### PR DESCRIPTION
**Pull request by redmoogle**
# Document the changes in your pull request
image

:cl:  Altoids1
spellcheck: Removed bad syntax
/:cl:

### CODE REVIEW

**Comment by Altoids1** [comment]
Stole my pr!!!! jerk!!!!!!!!!!!

**Comment by redmoogle** [comment]
I cannot read

File: code/modules/antagonists/clockcult/clock_mobs/_eminence.dm [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 //The Eminence is a unique mob that functions like the leader of the cult. It's incorporeal but can interact with the world in several ways.
 /mob/camera/eminence
-	name = "\the Emininence"
-	real_name = "\the Eminence"
+	name = "Emininence"
```
Comment (Altoids1) [inline code/modules/antagonists/clockcult/clock_mobs/_eminence.dm]: Emininence?

**Review by Altoids1 — CHANGES_REQUESTED** [review]

**Comment by redmoogle** [comment]
Actual author request

**Pull request closed without merging**